### PR TITLE
Users/aroberts/documentation

### PR DESCRIPTION
**Pull request by AngusWR**
Updated documentation and example code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,14 +29,73 @@ pip install --upgrade --no-index --find-links=. glasswall
 ```
 **Note:** The lxml whl includes the required lxml dependencies and has been tested on Centos 7 and RHEL8 environments.
```
Comment (mobrien-gw) [inline README.md:30]: What is whl?

Comment (mobrien-gw) [inline README.md:30]: Centos 7 --> CentOS 7

File: README.md [hunk 2]
```diff
@@ -89,10 +153,11 @@ glasswall.config.logging.console.setLevel(logging.DEBUG)
...
 ```
 
-</details>
+---
 
-<details>
-<summary>Content management policies</summary>
+### Content management policies
+
+Documentation about content management policies can be found here: https://docs.glasswall.com/docs/glasswall-engine-content-management
```
Comment (mobrien-gw) [inline README.md:160]: Documentation about content management policies can be found on the [Policy Management](https://docs.glasswall.com/docs/embedded-engine-policy-management) page.

File: README.md
Comment (mobrien-gw) [inline README.md:1]: Generally, I don't expose raw links to the reader.

**Review by mobrien-gw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AngusWR) [reply README.md:30]: wheel

File: tests/test_readme_python_code.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
+
+
+import os
+import subprocess
+import unittest
+
+
+def extract_python_blocks_from_readme(file_path):
+    # Function to extract Python blocks from the README and return as a dictionary
```
Comment (mobrien-gw) [inline tests/test_readme_python_code.py:9]: Should this be a docstring?

**Review by mobrien-gw — APPROVED** [review]

File: tests/test_readme_python_code.py
[the remark this answers is not in the document]
Comment (AngusWR) [reply tests/test_readme_python_code.py:1]: It's fine for internal use

File: tests/test_readme_python_code.py (hunk 3 above)
Comment (AngusWR) [reply tests/test_readme_python_code.py:9]: Internal, not really needing of a docstring

**Pull request merged**